### PR TITLE
Initial Lora Integration

### PR DESCRIPTION
**Pull request by YAS-CODE**
Just made a live feed of dummy Temperature sensor and soil sensor

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "bin": "./WRIO-InternetOS/bin/WRIO-InternetOS",
   "description": "https://github.com/webRunes/WRIO-InternetOS/",
   "private": true,
-  "repository": "https://github.com/webRunes/WRIO-InternetOS.git",
+  "repository": "https://github.com/YAS-CODE/WRIO-InternetOS.git",
```
Comment (AlexeyAnshakov) [inline package.json]: Wrong link

File: src/config.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
 var config = {};
 
-config.gatewaysServiceUrl = "https://imec.herokuapp.com";
+config.gatewaysServiceUrl = "http://54.89.128.49:5001";//"https://imec.herokuapp.com";
```
Comment (AlexeyAnshakov) [inline src/config.js]: Here must be a comment: if it's our AWS server - add this info

File: package.json (hunk 1 above)
Comment (YAS-CODE) [reply package.json]: Corrected It Now

File: src/config.js (hunk 2 above)
Comment (YAS-CODE) [reply src/config.js]: Added It up, Please take a look now

**Pull request merged**